src/background.js

```diff
@@ -1,5 +1,6 @@
 import OptionsSync from 'webext-options-sync';
-import injectContentScripts from 'webext-dynamic-content-scripts';
+import DPT from 'webext-domain-permission-toggle';
+import DCS from 'webext-dynamic-content-scripts';
```

Comment:
Don't use acronyms.

`DPT` => `domainPermissionToggle`
`DCS` => `dynamicContentScripts`

Reply:
@bfred-it Why the downvote? `DPT` isn't very readable. What's your argument for?

Reply:
I agree now, I've matured. 😌